### PR TITLE
[Dotnet/Samples] Add description for Tizen.CircularUI and move ImageR…

### PR DESCRIPTION
**Pull request by hj-na-park**
- Add description for Tizen.CircularUI
- Move ImageReader app to UI category

### CODE REVIEW

File: docs/application/dotnet/samples/account/overview.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 			<td><img alt="" height="180" src="media/waccounts.png" width="180"/></td>
 			<td>
 			<p><strong>(W) Accounts</strong> [In progress]</p>
-			<p>This sample application demonstrates how to implement an account provider which can add and configure an account.</p>
+			<p>This sample application demonstrates how to implement an account provider which can add and configure an account. It also demonstrates how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/account/overview.md]: done

File: docs/application/dotnet/samples/appfw/overview.md [hunk 2]
```diff
@@ -34,7 +34,7 @@
 			<td><img alt="" height="180" src="media/wappinformation.png" width="180"/></td>
 			<td>
 			<p><strong>(W) AppInformation</strong> [In progress]</p>
-			<p>This sample application demonstrates how to get the application-specific information such as AppId, Package ID, shared directory path. It also demonstrates how to get the directory information such as resource, cache, app data and so on.</p>
+			<p>This sample application demonstrates how to get the application-specific information such as AppId, Package ID, shared directory path. It also demonstrates how to get the directory information such as resource, cache, app data and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/appfw/overview.md]: done

File: docs/application/dotnet/samples/appfw/overview.md [hunk 3]
```diff
@@ -66,7 +66,7 @@
 			<td><img alt="" height="180" src="media/wappcontrol.png" width="180"/></td>
 			<td>
 			<p><strong>(W) AppControl</strong> [In progress]</p>
-			<p>This sample application demonstrates how to launch an application and get a result for the launch request. When you press <b>Launch</b> button, AppInformation sample application will be launched.</p>
+			<p>This sample application demonstrates how to launch an application, get a result for the launch request and create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>. When you press <b>Launch</b> button, AppInformation sample application will be launched.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/appfw/overview.md]: done

File: docs/application/dotnet/samples/network/overview.md [hunk 4]
```diff
@@ -14,36 +14,36 @@
...
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="180" src="media/wnetworkapp.png" width="180"/></td>
+			<td><img alt="" height="180" src="media/wnfc.png" width="180"/></td>
 			<td>
-			<p><strong>(W) NetworkApp</strong> [In progress]</p>
-			<p>This sample application demonstrates how to check info about your connection and manage your Wi-Fi settings.</p>
+			<p><strong>(W) NFCSampleApp</strong> [In progress]</p>
+			<p>This sample application demonstrates how to send and receive a NDEF message from remote Near Field Communication (NFC) device, and read NDEF message from NFC Tag. It also demonstrates how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/network/overview.md]: done

File: docs/application/dotnet/samples/network/overview.md [hunk 5]
```diff
@@ -14,36 +14,36 @@
...
 		</tr>
 		<tr>
-			<td><img alt="" height="267" src="media/m35nfc.png" width="150"/></td>
+			<td><img alt="" height="267" src="media/m17networkapp.png" width="150"/></td>
 			<td>
-			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NFCSampleApp" target="_blank"><strong>(M) NFC</strong></a></p>
-			<p>This sample application demonstrates how to work with Near Field Communication (NFC) API to send information to the nfc p2p device. It also demonstrates how to read and write the nfc tags.</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NetworkApp" target="_blank"><strong>(M) NetworkApp</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Network.Connection.html" target="_blank">Tizen.Network.Conncetion C# API</a>, Wi-Fi, and Wi-Fi Direct.<br />
+			This application provides network information such as IP address, network state, and list of available networks Using this information, you can connect to Wi-Fi.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/network/overview.md]: done

File: docs/application/dotnet/samples/network/overview.md [hunk 6]
```diff
@@ -14,36 +14,36 @@
...
 			</td>
 		</tr>
 		<tr>
-			<td><img alt="" height="180" src="media/wnfc.png" width="180"/></td>
+			<td><img alt="" height="180" src="media/wnetworkapp.png" width="180"/></td>
 			<td>
-			<p><strong>(W) NFCSampleApp</strong> [In progress]</p>
-			<p>This sample application demonstrates how to send and receive a NDEF message from remote Near Field Communication (NFC) device, and read NDEF message from NFC Tag.</p>
+			<p><strong>(W) NetworkApp</strong> [In progress]</p>
+			<p>This sample application demonstrates how to check info about your connection and manage your Wi-Fi settings. It also demonstrates how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/network/overview.md]: done

File: docs/application/dotnet/samples/system/overview.md [hunk 7]
```diff
@@ -35,7 +35,7 @@
 			<td><img alt="" height="180" src="media/wheartratemonitor.png" width="180"/></td>
 			<td>
 			<p><strong>(W) HeartRateMonitor</strong> [In progress]</p>
-			<p>This sample application demonstrates how to obtain data provided by the HRM sensor (heart rate sensor) using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a>.</p>
+			<p>This sample application demonstrates how to obtain data provided by the HRM sensor (heart rate sensor) using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a> and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
[the remark this answers is not in the document]
Comment (hj-na-park) [reply docs/application/dotnet/samples/system/overview.md]: done

**Comment by hj-na-park** [comment]
@sabeena21 

I fix all issues. Please check them.

File: docs/application/dotnet/samples/system/overview.md [hunk 8]
```diff
@@ -21,21 +21,21 @@
...
-			<p>This sample application demonstrates how to play sound and vibration feedback.</p>
+			<p>This sample application demonstrates how to play sound and vibration feedback and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>..</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m27heartratemonitor.png" width="150"/></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/HeartRateMonitor" target="_blank"><strong>(M) HeartRateMonitor</strong></a></p>
-			<p>This sample application demonstrates how to obtain data provided by the heart rate monitor (HRM) using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a>.</p>
+			<p>This sample application demonstrates how to obtain data provided by the Heart Rate Monitor (HRM) using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a>.</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/system/overview.md]: Hello, @sabeena21
In #349 or #324, I can see it was guided to place abbreviation in front of the word, as like "HRM (Heart Rate Monitor)". However, in this PR, abbreviation is guided to come later. Which rule are you going to use?

File: docs/application/dotnet/samples/system/overview.md [hunk 9]
```diff
@@ -21,21 +21,21 @@
...
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/wheartratemonitor.png" width="180"/></td>
 			<td>
 			<p><strong>(W) HeartRateMonitor</strong> [In progress]</p>
-			<p>This sample application demonstrates how to obtain data provided by the HRM sensor (heart rate sensor) using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a>.</p>
+			<p>This sample application demonstrates how to obtain data provided by the Heart Rate Monitor (HRM) sensor using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Sensor.html" target="_blank">Tizen.Sensor C# API</a> and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>.</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/system/overview.md]: Hello, @sabeena21
In #349 or #324, I can see it was guided to place abbreviation in front of the word, as like "HRM (Heart Rate Monitor)". However, in this PR, abbreviation is guided to come later. Which rule are you going to use?

File: docs/application/dotnet/samples/system/overview.md [hunk 10]
```diff
@@ -21,21 +21,21 @@
 			<td><img alt="" height="180" src="media/wfeedbackapp.png" width="180"/></td>
 			<td>
 			<p><strong>(W) FeedbackApp</strong> [In progress]</p>
-			<p>This sample application demonstrates how to play sound and vibration feedback.</p>
+			<p>This sample application demonstrates how to play sound and vibration feedback and how to create circular UI using <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI C# API</a>..</p>
```
Comment (mijin85cho) [inline docs/application/dotnet/samples/system/overview.md]: Please remove one duplicated period at the end.

File: docs/application/dotnet/samples/system/overview.md (hunk 8 above)
Comment (sabeena21) [reply docs/application/dotnet/samples/system/overview.md]: @mijin85cho, It is correct to write as "Heart Rate Monitor (HRM)"

To maintain consistency with reference to developer.tizen.org: https://developer.tizen.org/community/tip-tech/accessing-heart-rate-monitor-hrm-sensor-data-native-applications.

**Comment by hj-na-park** [comment]
@mijin85cho 
I fix all issues. Please check them.

**Comment by mijin85cho** [comment]
My comments are all fixed. so after @sabeena21 approve this PR, I'll merge it.

File: docs/application/dotnet/samples/multimedia/overview.md [hunk 11]
```diff
@@ -17,25 +17,6 @@
 			<p>This sample application demonstrates how to take a picture using <a href="https://developer.tizen.org/dev-guide/csapi/api/Tizen.Multimedia.html" target="_blank">Tizen. Multimedia C# API</a>.</p>
```
Comment (sabeena21) [inline docs/application/dotnet/samples/multimedia/overview.md]: Please can you confirm that whether it is Tizen. Multimedia C# API or Tizen.Multimedia C# API?

**Comment by hj-na-park** [comment]
@sabeena21 
I fix issue. Please check it.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**